New_PlatformGame/Assets/NewPlayerMovement.cs

```diff
@@ -19,8 +20,7 @@ public class NewPlayerMovement : MonoBehaviour
     //private 변수
     private Rigidbody2D _rigidbody2D;
     private Collider2D _collider2D;
-    private GameObject _jumpBar; //점프바 변수
-    private SpriteRenderer _jumpBarSprite; //점프바의 스프라이트 렌더러
+    public GameObject _jumpBar; //점프바 변수
```

Comment:
```suggestion
    public GameObject _jumpBar; //점프바 변수
```
이 역시 private에서 public으로 바뀐건 컨벤션에 맞춰 수정하도록 하자.